db/primary_migrate/20220902162411_add_allow_threatmetrix_to_service_providers.rb

```diff
@@ -0,0 +1,10 @@
+class AddAllowThreatmetrixToServiceProviders < ActiveRecord::Migration[7.0]
+  def up
+    add_column :service_providers, :allow_threatmetrix, :boolean
+    change_column_default :service_providers, :allow_threatmetrix, false
```

Comment:
I think setting a default value can be an expensive operation and lock the table. This is a hot table so I am suspicious of it. Luckily since it's a Boolean, defaulting to "null" is basically false and should be fine. I'd recommend we remove this

Reply:
it's normally expensive but it's a tiny table